pcsx2/Gif.cpp

```diff
@@ -553,27 +553,24 @@ static __fi bool mfifoGIFrbTransfer()
 	if (src == NULL)
 		return false;
 
-	u32 MFIFOUntilEnd = ((dmacRegs.rbor.ADDR + dmacRegs.rbsr.RMSK + 16) - gifch.madr) >> 4;
-	bool needWrap = MFIFOUntilEnd < qwc;
-	u32 firstTransQWC = needWrap ? MFIFOUntilEnd : qwc;
-	u32 transferred;
-
-	transferred = WRITERING_DMA((u32*)src, firstTransQWC); // First part
+	const u32 MFIFOUntilEnd = ((dmacRegs.rbor.ADDR + dmacRegs.rbsr.RMSK + 16) - gifch.madr) >> 4;
+	const bool needWrap = MFIFOUntilEnd < qwc;
+	const u32 firstTransQWC = needWrap ? MFIFOUntilEnd : qwc;
+	const u32 transferred = WRITERING_DMA((u32*)src, firstTransQWC); // First part
 
 	gifch.madr = dmacRegs.rbor.ADDR + (gifch.madr & dmacRegs.rbsr.RMSK);
 	gifch.tadr = dmacRegs.rbor.ADDR + (gifch.tadr & dmacRegs.rbsr.RMSK);
 
 	if (needWrap && transferred == MFIFOUntilEnd)
 	{
 		// Need to do second transfer to wrap around
-		u32 transferred2;
-		uint secondTransQWC = qwc - MFIFOUntilEnd;
+		const uint secondTransQWC = qwc - MFIFOUntilEnd;
```

Comment:
could be moved after the src check, just for scope/pointless initialisation sake.